multicast/recv.py

```
@@ -211,6 +212,82 @@
+	f"Will bind to {multicast._MCAST_DEFAULT_BIND_IP}.",
+	"Tip: Pass a value for bind_group to suppress this message",
+	"(such as 'bind_group=multicast._MCAST_DEFAULT_BIND_IP')",
+])
+
+_mcast_recv_join_only_multicast_warn_msg = "\n".join([
+	_mcast_recv_bind_not_join_prefix,
+	"Just use socket.Socket.bind(...) for non-multicast networking.",
+])
+
+
+def _validate_join_args(groups=None, port=None, iface=None, bind_group=None, isock=None):
```

Ruff (0.8.2)

Missing return type annotation for private function `_validate_join_args`

(ANN202)